exporter: Add autoplay MovieClip option to exporter.

AeroXuk opened the pull request:
Issue #20470 
Here is my attempt of add this functionality to ruffle as a PR.
I would be grateful if someone more familier with the codebase could review it and let me know if anything needs changing.

Also let me know if the way I've implemented this would have any unintended consiquences. For my purposes this change has worked and allowe me to export some old swfs into frames ready for conversion to mp4 via ffmpeg.

n0samu commented:
I took a quick look and this seems good to me, thank you! Be sure to run `cargo fmt` and amend your commit so the checks will pass, though.

AeroXuk commented:
> I took a quick look and this seems good to me, thank you! Be sure to run `cargo fmt` and amend your commit so the checks will pass, though.

Thank you, I've run the `cargo fmt` command and commited to my branch. Let me know if you want me to change anything else.

AeroXuk commented on exporter/src/main.rs:
```diff
@@ -64,6 +65,11 @@ struct Opt {
+    /// Automatically resume the root MovieClip if it becomes stopped.
+    /// Allows the bypassing of Click to Play buttons.
+    #[clap(long)]
+    autoplay: bool,
```
Good suggestion, I'll rename the CLI flag and update the message to better describe what it does and include a warning.

AeroXuk commented:
@kjarosh I've refactored the operations into a new function, added the extra usings, renamed the cli operator and clarified what it does with warning on the cli help.

kjarosh reviewed (approved):
LGTM, thank you!
